packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -691,6 +692,12 @@ class _SerializeConfig {
     includeProperties = base.includeProperties,
     expandPropertyValues = base.expandPropertyValues;
 
+  /// Optional object group name used to manage manage lifetimes of object
```

Comment:
Making groupName optional is added as part of this CL so I can get lightweight info on the Element tree without object ids for use by logic in IntelliJ to generate lint errors given rebuild counts and a snapshot of the widget tree to determine whether frequently built widgets had parents of specific types.